### PR TITLE
add view_message example

### PR DESCRIPTION
**Pull request by IT-Berater**
New commandline wraper option -t or --text. 

Simple use from command line.

### CODE REVIEW

**Comment by IT-Berater** [comment]
Fix blank line

File: examples/view_message.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    device = max7219(serial, cascaded=n or 1, block_orientation=block_orientation,
+                     rotate=rotate or 0, blocks_arranged_in_reverse_order=inreverse)
+    print(text)
+
+    show_message(device, text, fill="white", font=proportional(CP437_FONT), scroll_delay=0.05)
+    time.sleep(1)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='view_massage arguments',
```
Comment (thijstriemstra) [inline examples/view_message.py:28]: typo: "view_massage"

Comment (IT-Berater) [reply examples/view_message.py:28]: Thanks

**Comment by IT-Berater** [comment]
.

**Comment by thijstriemstra** [comment]
thanks!

**Pull request merged**